debian/autobake-deb.sh

```diff
@@ -170,8 +170,21 @@ BUILDPACKAGE_DPKGCMD=()
 # Fakeroot test
 if fakeroot true; then
   BUILDPACKAGE_DPKGCMD+=( "fakeroot" "--" )
+
+  echo "Testing if 'fakeroot' command is working with '/usr/bin/true'"
```

Comment:
'/usr/bin/true' should be '/bin/true'. Or, adjust line 177 to be '/usr/bin/true'

This log is likely unnecessary. A user does not need to know that the 'fakeroot' command is being tested for existence. Instead, the error output on line 180 could be more specific such as "Failed to execute 'fakeroot -- /bin/true'. Proceeding without 'fakeroot'"

Reply:
I've update it back to `/bin/true` as it's preset location. Debian is going thru Fedora and others everything in `/usr/bin/` transition so that's why it was `/usr/bin/true`